vncdotool/command.py

```diff
@@ -398,6 +398,9 @@ def vncdo():
         metavar='FACTOR', default=1.0,
         help='pause time is accelerated by FACTOR [x%default]')
 
+    op.add_option('-i', '--incremental-refreshes', action='store_true', default=False,
+        help='set the "incremental" flag')
```

Comment:
I always struggle with these flag names.  I'd like to make them more approachable to users and hide VNC protocol details but that can also make it harder to understand.  I guess my real question is what is the correct default, what is the most compatible setting we can use that works for most people out of the box?

Reply:
The current default value is 0 (False) so I kept it the same.
Regarding the flag name, I think that it needs to give enough information on what it is used for, I can change it if you have a better idea.